app/src/main/java/openfoodfacts/github/scrachx/openfood/features/HomeFragment.kt

```diff
@@ -154,31 +154,30 @@ class HomeFragment : NavigationBaseFragment() {
 
     override fun onResume() {
         super.onResume()
-        val productCount = sharedPrefs.getInt(PRODUCT_COUNT_KEY, 0)
-
-        refreshProductCount(productCount)
+        refreshProductCount()
         refreshTagLine()
 
         (activity as? AppCompatActivity)?.supportActionBar?.let { it.title = "" }
     }
 
-    private fun refreshProductCount(oldCount: Int) {
+    private fun refreshProductCount() {
         Log.d(LOG_TAG, "Refreshing total product count...")
-
-        lifecycleScope.launch {
-            val resp = try {
-                withContext(Dispatchers.IO) { productsApi.getTotalProductCount(getUserAgent()) }
+        lifecycleScope.launchWhenResumed {
```

Comment:
main fix
based on log "at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)" I can assume, that the coroutine resumed the work while binding is still not inflated